skops/hub_utils/_hf_hub.py

```diff
@@ -9,9 +9,17 @@
 from pathlib import Path
 from typing import List, Union
 
+import numpy as np
 from huggingface_hub import HfApi, snapshot_download
 from requests import HTTPError
 
+SUPPORTED_TASKS = [
+    "tabular-classification",
+    "tabular-regression",
+    "text-classification",
+    "text-regression",
```

Comment:
What is `text-regression`? We don't have such task in Hub, no? 
<img width="347" alt="Ekran Resmi 2022-07-21 13 11 53" src="https://user-images.githubusercontent.com/53175384/180200325-318e84aa-25d9-4302-a3c7-308a35437a30.png">

Reply:
Related issue from `hub-docs` for visibility https://github.com/huggingface/hub-docs/issues/137. 

The issue has some complexity for `text-regression` since as of now `transformers` uses `text-classification` to do both classification and regression, and splitting it now would be complex (as per https://github.com/huggingface/hub-docs/issues/137#issuecomment-1122859453). The "consistent" approach for the Hub would be using `text-classification` for both tasks. 

cc @narsil @julien-c

Reply:
I don't think we have to change anything on the transformers side, and those users might be happy with calling regression classification, but I don't think a sklearn pipeline which has a `LinearRegression` at the end of it would be a good fit for a `text-classification` task. As we also talked about it in the issue you mention, from the statistical inference / ML perspective, classification and regression are not really the same thing.

Reply:
We also don't have to fully support it now. Users can set the tag and certain things, like inference wouldn't work, and they can hack around and use the workaround of calling a regression classification to get a widget if they want to, and later we can fix the issue.

Reply:
Yes, that sounds like a good path forward for me so we can move this forward, but I'll think a bit about this topic as the point you make is totally valid, and **it will** be confusing for those users. I think this might be solved via an alias as discussed in the issue, but let's think a bit more about this

Reply:
Purely from the `transformers` side we can definitely add an alias. https://github.com/huggingface/transformers/blob/main/src/transformers/pipelines/__init__.py#L128

Also there's an open issue (with not much activity) to get actually text-regression in addition to text-classification.

Reply:
The consensus seems to be that we'll add `text-regression`, either as an alias or a task, but not a prerequisite for this PR. Therefore resolving the discussion. Thanks for all the comments.